ideascube/mediacenter/templates/mediacenter/document_detail.html

```diff
@@ -18,7 +18,7 @@
     {% elif document.preview %}
         <a href="{% media document 'original' %}"><img src="{% media document 'preview' %}" /></a>
     {% endif %}
-    <div class="text">{{ document.summary }}</div>
+    <div class="text">{{ document.summary|safe }}</div>
```

Comment:
TinyMCE interprets all the text the user enter a plain text and escape it.
It's tinyMCE itself that add the `p`, `a`, `img` elements.

> What happens if I disable Javascript and then manually enter some malicious HTML?
This is a problem indeed. But it is not related at all with js or tinyMCE.
A user could just send itself a form post with malicious value in the summary input. As we do not check anything on django side (and I don't know how because we want to handle html coming from tinyMCE) this is indeed a security issue.